org.catacombing.kumo.json

```diff
@@ -0,0 +1,120 @@
+{
+    "id": "org.catacombing.kumo",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "24.08",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.rust-stable",
+        "org.freedesktop.Sdk.Extension.llvm20"
+    ],
+    "command": "kumo",
+    "finish-args": [
+        "--socket=pulseaudio",
+        "--socket=wayland",
```

Comment:
Not providing X11 support seems to cause a warning, however my application **intentionally** does not support X11 to simplify development. It was mentioned that some lints can be ignored, but I wanted to make clear that this was not omitted by accident.

Reply:
you need to submit an exception then https://docs.flathub.org/docs/for-app-authors/linter#exceptions

https://docs.flathub.org/docs/for-app-authors/linter#finish-args-only-wayland